pkg/controller/controller_test.go

```diff
@@ -2838,6 +2840,47 @@ func TestDelete(t *testing.T) {
 			},
 			expectErr: true,
 		},
+		"fail - delete when attached to node": {
```

Comment:
Can you add 2 more test case like this, except:
- the VA.status.attached = false
- the VA has a deletion timestamp on it

Both of these cases should still behave as if the volume is still attached.

Reply:
done!